user_tools/README.md

```diff
@@ -35,12 +35,15 @@ Set up a Python environment with a version between 3.8 and 3.10
       $ pip install -e .
       ```
     - Using wheel package built from the repo
-
       ```sh
-      $ pip install build
-      $ python -m build --wheel
+      $ ./build.sh
```

Comment:
The build command will fail if the `spark_rapids_pytools` package is not installed.

The instructions needs to be re-structured. In the installation part.
A user only needs `pip install <wheel-file>` to install the tools.
Then we create a section/subsection on how to build a wheel-file from source. This will include the `offline` mode which avoids downloading resources from internet.

```
pip install -e .
./build.sh
```

then the offline part

```
pip install -e .
./build.sh offline
```

Reply:
Updated the README in two sections